apps/nextjs-web/app/components/info-card/feedback-buttons.tsx

```diff
@@ -1,41 +1,52 @@
 'use client'
 
+import { Enum_Feedback } from '@/src/gql/graphql'
 import Image from 'next/image'
 import { useState } from 'react'
 
-type Feedback = 'up' | 'down' | undefined
-
-export const FeedbackButtons = ({ feedback }: { feedback: Feedback }) => {
-  const [feedbackChoice, setFeedbackChoice] = useState<Feedback>(feedback)
+export const FeedbackButtons = ({
+  feedback,
+}: {
+  feedback?: Enum_Feedback | null
+}) => {
+  const [feedbackChoice, setFeedbackChoice] = useState<
+    Enum_Feedback | undefined | null
```

Comment:
In `title.tsx` this component is rendered with null coalescing:
```tsx
<FeedbackButtons feedback={feedback ?? undefined} />
```

```suggestion
  feedback?: Enum_Feedback
}) => {
  const [feedbackChoice, setFeedbackChoice] = useState<
    Enum_Feedback | undefined
```